### ISSUE TITLE
Bogus `Until:` is printed by `systemctl status` for every unit type

### ISSUE THREAD

**Issue by arvidjaar** [systemd/systemd#26473]
### systemd version the issue has been seen with

252.5

### Used distribution

openSUSE Tumbleweed

### Linux kernel version used

6.1.10-1-default

### CPU architectures issue was seen on

x86_64

### Component

systemctl

### Expected behaviour you didn't see

`Until:` is printed only for units of type `service` and `scope` if the value is not default.

### Unexpected behaviour you saw

`Until:` with `RuntimeMaxSec == 0` is printed for every unit type.

### Steps to reproduce the problem

Just run `systemctl status`

### Additional program output to the terminal or log subsystem illustrating the issue

```sh
bor@tw:~> systemctl status network.target 
\u25cf network.target - Network
     Loaded: loaded (/usr/lib/systemd/system/network.target; static)
     Active: active since Sat 2023-02-18 11:26:31 MSK; 9min ago
      Until: Sat 2023-02-18 11:26:31 MSK; 9min ago
       Docs: man:systemd.special(7)
             https://systemd.io/NETWORK_ONLINE

Warning: some journal files were not opened due to insufficient permissions.
bor@tw:~>
```

**Comment by yuwata** [comment]
Hm? Could you show `systemctl show -p RuntimeMaxUSec UNIT` ?

**Comment by yuwata** [comment]
Ah, the issue is for e.g. target unit.

**Comment by yuwata** [comment]
Fix is waiting in #26534.
